streams/src/test/java/org/apache/kafka/streams/processor/internals/StandbyTaskTest.java

```diff
@@ -323,8 +505,34 @@ public void shouldNotThrowUnsupportedOperationExceptionWhenInitializingStateStor
         final InternalTopologyBuilder internalTopologyBuilder = InternalStreamsBuilderTest.internalTopologyBuilder(builder);
         final ProcessorTopology topology = internalTopologyBuilder.setApplicationId(applicationId).build(0);
 
-        new StandbyTask(taskId, partitions, topology, consumer, changelogReader, config,
+        final StandbyTask standbyTask = new StandbyTask(taskId, partitions, topology, consumer, changelogReader, config,
+            new MockStreamsMetrics(new Metrics()), stateDirectory);
+
+        standbyTask.initializeStateStores();
```

Comment:
This is actually where the exception would get thrown. We weren't actually testing it.